mono/mini/TestDriver.cs

```diff
@@ -13,10 +13,40 @@ public class CategoryAttribute : Attribute
 		get; set;
 	}
 }
+public class TestDriverReporter
+{
+	private int failedTests = 0;
+	private int skippedTests = 0;
+	private int executedTests = 0;
+
+	public int FailedTests {
+		get {
+			return failedTests;
+		}
+	}
+
+	public int SkippedTests {
+		get {
+			return skippedTests;
+		}
+	}
+
+	public int ExecutedTests {
+		get {
+			return executedTests;
+		}
+	}
```

Comment:
This could be simplified to:

```cs
public int FailedTests { get; private set; }
public int SkippedTests { get; private set; }
public int ExecutedTests { get; private set; }
```